### DISCUSSION TITLE
Why is `Conversation View` listed as advantage over Proton despite its workitem still being open and it still being on the Roadmap?

### DISCUSSION THREAD

**Discussion by KernelBypass** [tutao/tutanota#5767] [General]
Here `conversation view` is listed as a ready-to-use advantage/pro of Tutanota over Proton:
https://tutanota.com/protonmail

![image](https://github.com/tutao/tutanota/assets/62620821/003fb735-2a03-42ed-a056-2919a62c5c7b)

Despite it still being an open Roadmap item (#6) here:
https://tutanota.com/roadmap

And an open 9-year old workitem here:
https://github.com/tutao/tutanota/issues/6

Which one is it?

**Comment by vitoreiji** [comment]
Conversation view is already available, but the issue is still open because we still have ideas to further improve it :-)
